yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -255,13 +257,17 @@ func (conv *DebeziumValueConverter) GetTableNameToSchema() *utils.StructMap[sqln
 		colSchemaMap := make(map[string]map[string]string)
 
 		for _, col := range tblSchema.Columns {
-			colSchemaMap[col.Name] = col.Schema.Parameters
+			colNameQuoted, err := conv.tdb.QuoteAttributeName(tbl, col.Name)
+			if err != nil {
+				return false, fmt.Errorf("error quoting attribute name %s for table %s: %w", col.Name, tbl, err)
+			}
+			colSchemaMap[colNameQuoted] = col.Schema.Parameters
```

Comment:
Are you sure we need quoted column names inside value converter? I guess not. COPY statement (during snapshot load) and IUD statements during streaming are the only two places where we need to quote the column names.

Reply:
Yeah, a test case failed because of this. 
basically, we are storing a map[column]schema. and we lookup in this map while importing. Now, the lookup key was quoted, but this map had unquoted column names as keys. That's why I had to make this also quoted.